views/partials/_category-nav.njk

```diff
@@ -4,7 +4,8 @@ in this section.
 This is mostly to support non-JS browsing on a mobile viewport, where the 'top'
 navigation doesn't expand and so the user can only access the top level category
 pages. #}
-<nav class="category-nav">
+<!-- [html-validate-disable-block unique-landmark -- allow 2 navs with the same aria-labelledby because they contain the same information] -->
```

Comment:
What happens if we only add this to the 'second' in-page nav? If that worked, I think that'd mean we can avoid needing to disable the `no-unused-disable` rule (which means we're more likely to clean up after ourselves if we fix this later.

If we need both, could we use `disable-next` instead of `disable-block`, or does that not work here?

Reply:
Unfortunately it still kicks off. Without the ignore on the 'main' sidebar, html-validate gets upset on landing pages because then that main sidebar is copying the second sidebar's `aria-labelledby`.

Swapping `block` for `next` does work though so I've done that.

Reply:
No worries, I wondered if that might be the case. Thanks for checking!